src/content/docs/dropins/payment-services/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Payment Services overview
+description: Learn about the features of the Payment Services drop-in component.
+---
+
+import Link from '@components/Link.astro';
+import List from '@components/List.astro';
+
+The Payment Services drop-in renders the credit card form, allowing shoppers to efficiently enter their payment details. The drop-in currently supports credit/debit cards only.
+
+## Available Payment Services
```

Comment:
I don't understand what "Available Payment Services" means. What is "a payment service"? If it lists containers, why don't we call it "Available containers"?

Reply:
we can use **available containers**, but Payment Services is the "official" name of the extension/service/drop-in